code/datums/world_topic.dm

```diff
@@ -1,10 +1,10 @@
 // SETUP
 
-/proc/TopicHandlers()
+/proc/TopicHandlers(var/datum/A)

```

Comment:
If you feed this a typepath then the variable wouldn't be of type /datum. Just do var/datum imo.

Reply:
Understood.